packages/documentation/src/pages/getting-started/common-tasks/new-end-to-end-test.mdx

```diff
@@ -23,124 +24,50 @@ Front end engineers use end-to-end (e2e) tests in `vets-website` to validate mul
 
-_Command commands are available on the Nightwatch client e.g. `client.axeCheck()`. Below are some of the commonly used custom Nightwatch commands._
+Below are some of the commonly used custom Nightwatch commands (accessible from the link above).\_
 
-- [`axeCheck(selector)`](https://github.com/department-of-veterans-affairs/vets-website/blob/d721cf0581c6b7d6f26903540dfc27f26a16a7be/src/platform/testing/e2e/nightwatch-commands/axeCheck.js#L4-L7) - uses the [`axe-core`](https://github.com/dequelabs/axe-core/blob/master/doc/developer-guide.md) library to run a series of tests to check for accessibility of content and functionality for DOM nodes in the selector
-- [`clickIf(selector, predicate, ...predicateArguments)`](https://github.com/department-of-veterans-affairs/vets-website/blob/d721cf0581c6b7d6f26903540dfc27f26a16a7be/src/platform/testing/e2e/nightwatch-commands/clickIf.js#L4-L7) - clicks the input at the selector when the predicate returns true
-- [`fill(selector, value, callback)`](https://github.com/department-of-veterans-affairs/vets-website/blob/d721cf0581c6b7d6f26903540dfc27f26a16a7be/src/platform/testing/e2e/nightwatch-commands/fill.js#L5-L6) - clears the current value and sets to the value provided
-- [`fillAddress(baseName, address}`)](https://github.com/department-of-veterans-affairs/vets-website/blob/d721cf0581c6b7d6f26903540dfc27f26a16a7be/src/platform/testing/e2e/nightwatch-commands/fillAddress.js#L2-L5) - fills an address widget with the value provided
-- [`fillCheckbox(selector, predicate, ...predicateParams`)](https://github.com/department-of-veterans-affairs/vets-website/blob/343d77e2d7509cdcecee4b41c723d01ca0147881/src/platform/testing/e2e/nightwatch-commands/fillCheckbox.js#L4-L7) - clicks the checkbox at the selector when the predicate returns true
-- [`fillDate(fieldName, dateString}`)](https://github.com/department-of-veterans-affairs/vets-website/blob/d721cf0581c6b7d6f26903540dfc27f26a16a7be/src/platform/testing/e2e/nightwatch-commands/fillDate.js#L4-L6) - fills a date widget at the fieldName with the dateString provided e.g. 1990-1-28
-- [`openUrl(url, disableForesee = true)`](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/e2e/nightwatch-commands/openUrl.js) - navigates to url and disables user feedback module (this can interfere with end-to-end tests)
-- [`selectRadio(fieldName, value)`](https://github.com/department-of-veterans-affairs/vets-website/blob/d721cf0581c6b7d6f26903540dfc27f26a16a7be/src/platform/testing/e2e/nightwatch-commands/selectRadio.js#L2-L6) - selects the provided option on yesNo widget at the field name
-- [`selectYesNo(fieldName, predicate)`](https://github.com/department-of-veterans-affairs/vets-website/blob/d721cf0581c6b7d6f26903540dfc27f26a16a7be/src/platform/testing/e2e/nightwatch-commands/selectYesNo.js#L2-L6) - selects the provided options on yesNo widget at the field name when the predicate returns true
+- axeCheck
```

Comment:
Since the previous docs included links to the source of the custom commands, I think it would be nice to have links to the source of the Cypress custom commands. 

That being said, I won't reject this PR for omitting those links.